.github/workflows/ci.yml

```diff
@@ -24,5 +24,6 @@ jobs:
         py.test -vv --cov=injector --cov-branch --cov-report html --cov-report term
         if which mypy; then mypy injector ; fi
         if which black; then black --check . ; fi
+        if which check-manifest; then check-manifest ; fi
```

Comment:
No conditional here please, since the dependency is unconditional

Reply:
fixed